adzerk/transform.py

```diff
@@ -59,6 +64,34 @@ def tracking_url_to_shim(url):
     return ','.join([path_id,e,s])
 
 
+def is_collection(spocs):
+    """
+    :param spocs: A list of spocs
+    :return: True if the list of spocs is a sponsored collection; spocs that should be featured together.
+    """
+    return all(spoc.get('collection_title') for spoc in spocs)
```

Comment:
We talked briefly about what we should if there are < 3 spocs in a collection. Is this a good place to validate and decide what to do?

Reply:
Good question, but I would argue that's a requirement that should be met client-side.